OrbitQt/orbitmainwindow.cpp

```diff
@@ -116,13 +116,18 @@ OrbitMainWindow::OrbitMainWindow(QApplication* a_App, ApplicationOptions&& optio
       [this](DataView* callstack_data_view, std::shared_ptr<SamplingReport> report) {
         this->OnNewSamplingReport(callstack_data_view, std::move(report));
       });
+
+  ui->RightTabWidget->setTabEnabled(ui->RightTabWidget->indexOf(ui->selectionTab), false);
```

Comment:
Not really related to this PR and a nitpick, but should we rename the "selectionTab" to sampling like "seletionSamplingTab" in analogy to "topDownTab" and "selectionTopDownTab"?

Reply:
Gladly done as it was bothering me as well. By the way, we should do some general renaming in `orbitmainwindow.ui` to at least use only one casing style (`centralWidget` vs `RightTabWidget` vs `capture_toolbar`...).